Jenkinsfile

```diff
@@ -5,10 +5,7 @@ env.CONCURRENCY = 10
 
 
 node('vetsgov-general-purpose') {
-  properties([[$class: 'BuildDiscarderProperty', strategy: [$class: 'LogRotator', daysToKeepStr: '60']],
-              parameters([choice(name: "cmsEnvBuildOverride",
-                                 description: "Choose an environment to run a content only build. Select 'none' to run the regular pipeline.",
-                                 choices: ["none", "dev", "staging"].join("\n"))])]);
+  properties([[$class: 'BuildDiscarderProperty', strategy: [$class: 'LogRotator', daysToKeepStr: '60']]]);
```

Comment:
I wonder if we should leave it so that the existing way of trigger content deploys works, but still make the changes to the content job that we want them to use going forward? That might make for a smoother transition, because I don't know how quickly they could get updates to their build trigger into dev and staging.